recipes/human-learn/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "human-learn" %}
+{% set version = "0.2.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/human-learn-{{ version }}.tar.gz
+  sha256: 7d7a1372428b7377c3aad20e62e5d0de0dc3ba1379f4c83f0290e4281c0a6d12
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - bokeh >=2.2.1
+    - clumper >=0.2.5
+    - pandas >=0.23.4
+    - python >=3.6
+    - scikit-learn >=0.23.2
+    - shapely >=1.7.1
+
+test:
+  imports:
+    - hulearn
+    - hulearn.classification
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/human-learn/
+  summary: natural intelligence benchmarks for scikit-learn
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add comment:

```yaml
# license file manually packaged: https://github.com/koaning/human-learn/pull/42
license_file: LICENSE
...
```

I put in a PR upstream to fix having to manually package the license file (https://github.com/koaning/human-learn/pull/42).